tests/trustyai/drift/utils.py

```diff
@@ -242,3 +251,34 @@ def _check_pods_ready_with_env() -> bool:
     for sample in samples:
         if sample:
             return
+
+
+def verify_metric_request(
+    client: DynamicClient, trustyai_service: TrustyAIService, token: str, metric_name: str, json_data: Any
+) -> None:
+    """
+    Sends a metric request to a TrustyAIService and validates the response.
+
+    Args:
+        client (DynamicClient): The client instance for interacting with the cluster.
+        trustyai_service (TrustyAIService): The TrustyAI service instance to interact with.
+        token (str): Authentication token for the service.
+        metric_name (str): Name of the metric to request.
+        json_data (Any): JSON payload for the metric request.
+    """
+
+    response = TrustyAIServiceRequestHandler(token=token, service=trustyai_service, client=client).send_drift_request(
+        metric_name=metric_name, json=json_data
+    )
+    LOGGER.info(msg=f"TrustyAI metric request response: {json.dumps(json.loads(response.text), indent=2)}")
+    response_data = json.loads(response.text)
+
+    assert response.status_code == http.HTTPStatus.OK, f"Unexpected status code: {response.status_code}"
+    assert response_data["timestamp"] != "", "Timestamp is empty"
+    assert response_data["type"] == "metric", "Incorrect type"
+    assert response_data["value"] != "", "Value is empty"
+    assert isinstance(response_data["value"], float), "Value must be a float"
+    assert response_data["specificDefinition"] != "", "Specific definition is empty"
+    assert response_data["name"] == metric_name, f"Wrong name: {response_data['name']}, expected: {metric_name}"
+    assert response_data["id"] != "", "ID is empty"
+    assert response_data["thresholds"] != "", "Thresholds are empty"
```

Comment:
aggregate failures and assert once; this way you get all failures and not exit on the 1st one

Reply:
done